app/models/version.rb

```diff
@@ -204,18 +209,21 @@ def info
   end
 
   def update_attributes_from_gem_specification!(spec)
-    update!(
-      authors: spec.authors,
-      description: spec.description,
-      summary: spec.summary,
-      licenses: spec.licenses,
-      metadata: spec.metadata || {},
-      requirements: spec.requirements,
-      built_at: spec.date,
-      required_rubygems_version: spec.required_rubygems_version.to_s,
-      required_ruby_version: spec.required_ruby_version.to_s,
-      indexed: true
-    )
+    transaction do
+      update!(
+        authors: spec.authors,
+        description: spec.description,
+        summary: spec.summary,
+        licenses: spec.licenses,
+        metadata: spec.metadata || {},
+        requirements: spec.requirements,
+        built_at: spec.date,
+        required_rubygems_version: spec.required_rubygems_version.to_s,
+        required_ruby_version: spec.required_ruby_version.to_s,
+        indexed: true
+      )
+      rubygem.update!(indexed: true)
```

Comment:
Why are we not updating this in `Rubygems#update_attributes_from_gem_specification!`?

Reply:
I put it in here since the versions indexed column is set to true in here as well to keep it together.

Reply:
Sorry but I don't understand why do we want to keep update to index column in the same function. Can you please explain this? Shouldn't we keep update to rubygem's attributes in the rubygems model if possible?
Order of update queries will remain the same if we update rubygems's indexed after `update_versions!` in `Rubygems#update_attributes_from_gem_specification!` and we won't need to add transaction here. You have added a hook on `indexed` column, so I don't think we would ever miss updates.

Reply:
If you think model hook is enough, I can remove this.

Reply:
removed and rebased